### PR TITLE
Restructure

### PR DESCRIPTION
**Pull request by shikhanansi**
Hi,
I have restructured the docs to remove the deep-nesting and give it an easy-to-navigate, flatter structure.
Kindly review the layout and provide feedback.
Regards,
Shikha

P.S. @ephox-mogran during this exercise, I have heavily restructured the Context Toolbars and Context Forms sections. Please review them and let me know if you have any concerns.

### CODE REVIEW

**Comment by shikhanansi** [comment]
> Just to let you know, to support old URLs that people posted in stackoverflow, for every change in documentation urls we will have to add redirect rules to our [nginx config](http://stash/projects/DX/repos/tiny.cloud-infrastructure/browse/vars/environments/shared/nginx-config.yml).
> 
> Let have a chat about that when you will have time

[JIRA task](https://ephocks.atlassian.net/browse/DOC-151) created to update the links redirect [file ](http://stash/projects/DX/repos/tiny.cloud-infrastructure/browse/vars/environments/shared/nginx-config.yml).
Regards,
Shikha

File: _includes/context/priority.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+There are two settings that determine determine the priority: `predicate` and `scope`. The priority system mirrors the old [inlite](https://www.tiny.cloud/docs/themes/inlite/#quicklink) plugin from TinyMCE 4. The `predicate` is a function that takes in the current context position and returns a boolean. The `scope` is either `node` or `editor`. The whole priority process works as follows:
```
Comment (ephox-mogran) [inline _includes/context/priority.md]: In TinyMCE4, inlite was a theme. In TinyMCE5, it's a plugin. All references to inlite from TinyMCE4 need to say theme.

File: ui-elements/contextform.md [hunk 2]
```diff
@@ -2,29 +2,48 @@
...
 | ---- | ------- |
 | `launch` | This is the specification for the launching button that can appear in a ContextToolbar only. It will be either type: `contextformbutton` or `contextformtogglebutton`, and will be identical to those definitions below except it will **not** have an `onAction`. |
 
+#### Launching ContextForms from a ContextToolbar
+
+If a registered ContextForm has a `launch` setting, then it can be launched from a ContextToolbar. The name of item will be `form:${name}` (e.g. 'form:link'). When the user presses this button, the toolbar will change into the specified ContextForm. If you the user presses Esc in a ContextForm that was launched through a ContextToolbar, they will be returned to the original ContextToolbar.
+
+#### Launching a ContextForm programmatically
+
+There is an `editor` event called `contexttoolbar-show` that can be fired to show a ContextForm at the current selection. The event takes a parameter `toolbarKey` which specifies the name of the registered ContextForm or ContextToolbar to show.
```
Comment (ephox-mogran) [inline ui-elements/contextform.md]: We are likely to change this event name, as it isn't all that consistent with other event names.

File: ui-elements/contexttoolbar.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+{% include context/positioning.md %}
+
+
+<!--- ### Example Configuration
+
+In this example, a custom toolbar object with two custom toolbar buttons is created and added to a configuration object via the ui toolbar property. This config is then used to create an  editor by replace.
+
+```js
+var customToolbar = {
```
Comment (ephox-mogran) [inline ui-elements/contexttoolbar.md]: Same thing as with the Context Form ... this event name is likely to change. Sorry :)

File: ui-elements/contextform.md (hunk 2 above)
Comment (shikhanansi) [reply ui-elements/contextform.md]: I will track that in the final release document.

**Pull request merged**